### ISSUE TITLE
NextCloud not managing SSL Certificate correctly

### ISSUE THREAD

**Issue by caitken000** [truenas/charts#1902]
Hi - I think this is BUG.

I'm running:
Truenas: TrueNAS-SCALE-23.10.0.1
NextCloud (Official): 28.0.0 Chart version 1.6.54
Nginx (Official) 2..10.4 Chart version 1.0.21 

I have the following setup
Cloudflare -> Router (ports 80,443) -> Nginx -> (443) Truenas (https SSL works!) plus I have Nginx -> (9001) NextCloud (https broken!) 

I have the same Cloudflare certificate on Truenas and NextCloud

I have tested the Certificate at SSLtest and it gives an A+ rating.

I get a CERT AUTHORITY error when attempting to access the NextCloud:9001 login page.

I'm accessing both truenas and nextcloud from the same browser instance.

**Comment by caitken000** [comment]
I can verify that the current Truenas Official NextCloud does not install the SSL Certificate correctly.
I have installed the same Cloudflare Origin Certificate on both Truenas and Nextcloud and sucessssfully installed Nginx to forward SSL traffic to Truenas and Nextcloud.  Under this configuration Truenas is accessible securly via https, while Nextcloud is not.  Attempting to access Nextcloud results in a CERT AUTHORITY ERROR  in the browser.  I have checked OS time etc and have eliminated any of the obvious causes for this error.  Given the set up is identical between Truenas and Nextcloud I think this suggests there is something not configured correctly in the official Truenas/Nginx/Postgres bundle.
